atomic.go

```diff
@@ -20,7 +20,7 @@
 
 // Package atomic provides simple wrappers around numerics to enforce atomic
 // access.
-package atomic
+package atomic // import "go.uber.org/atomic"
```

Comment:
Are you sure you want the canonical path set? This will force users to use this import path.